rust/repositories.bzl

```diff
@@ -87,10 +87,13 @@ filegroup(
 
 filegroup(
     name = "rustc_lib",
-    srcs = glob([
+    srcs = glob(
+        [
         "lib/*{dylib_ext}",
```

Comment:
Nit: Can you indent these four more spaces (so they're indented from the square brackets).

Reply:
Indented